components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/AuthenticationContextCache.java

```diff
@@ -126,6 +173,25 @@ public AuthenticationContextCacheEntry getValueFromCache(AuthenticationContextCa
             // Update the cache again with the new value.
             super.addToCache(key, entry);
         }
+        if (entry != null) {
+            try {
+                AuthenticationContextLoader.getInstance().loadAuthenticationContext(entry.getContext());
+            } catch (SessionDataStorageOptimizationClientException e) {
+                if (log.isDebugEnabled()) {
+                    log.debug(String.format("Client error occurred while loading optimized authentication context"
```

Comment:
Let's remove string format here. As it is only 1 param and also since its the last param

Reply:
Resolved with [`87d8c01`](https://github.com/wso2/carbon-identity-framework/pull/4446/commits/87d8c01eaa49ef513f71a55b77154408818a31b3)